apps/courses/tests/test_cms_toolbars.py

```diff
@@ -1,59 +1,27 @@
 """
-Test suite of the toolbar extension for course pages
```

Comment:
This refactor factorizes the test so it can be run for organizations and courses without writing duplicate code.